reactive/coredns.py

```diff
@@ -47,3 +81,34 @@ def start_charm():
 def update_image():
     # handle a new image resource becoming available
     clear_flag('charm.coredns.started')
+
+
+@when('charm.coredns.started', 'coredns.joined')
+def send_ip():
+    """
+    Send CoreDNS IP to kuberentes-worker
+    """
+    try:
+        coredns = endpoint_from_flag('coredns.joined')
+        if coredns:
+            service_ip = get_service_ip('coredns')
+            if service_ip:
+                coredns.send_ip(service_ip)
+                clear_flag('coredns.joined')
```

Comment:
It might be a good idea to have a log here that says something like "Unable to find coredns service IP. Will try next hook run." or something like that to let the user know that things are going. This would be a good thing to put into a status message so you could see it in `juju status` while you're waiting for things to settle.